autolab/Autolab
# Customized MIME type check for submission files

## Description
This PR fixes an issue with customized MIME type check method for submission files and adds the documentation for this feature.

## Motivation and Context
1. The feature is not documented. Instructors aren't necessarily aware of it when they want to block students from submitting the wrong types of files.
2. The feature on the current master won't work due to an attempt to concatenate a string and an uninitialized `flash[:error]`.

## How Has This Been Tested?
1. Customize `checkMimeType` for an assessment. Refer to the updated documentation in this branch.
2. Submit a file that violates the rule. You should be able to see the error message displayed accordingly.
3. Submit a file that does not violate the rule. It should be successfully submitted.

## Types of changes


## Checklist:


## Other issues / help required




If unsure, feel free to submit first and we'll help you along.
